cloudflare/resource_cloudflare_load_balancer.go

```diff
@@ -0,0 +1,335 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/pkg/errors"
+)
+
+func resourceCloudFlareLoadBalancer() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudFlareLoadBalancerCreate,
+		Read:   resourceCloudFlareLoadBalancerRead,
+		Update: resourceCloudFlareLoadBalancerUpdate,
+		Delete: resourceCloudFlareLoadBalancerDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudFlareLoadBalancerImport,
+		},
+
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"zone": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"zone_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false, // allow reusing same load balancer for different dns names
```

Comment:
`ForceNew` is `false` by default, so this can be omitted. I am curious about what the comment means. It reads as if Load Balancer names are not unique? I'm not sure what `ForceNew` would influence there.

Reply:
I think initially I thought that this name was immutable for some reason, which it isn't. The actual behaviour is as you would normally expect, so I'm OK to delete this comment